_partials/windows_sqlite.md

```diff
@@ -0,0 +1,18 @@
+## SQLite
```

Comment:
When our the content is identical between Windows and Ubuntu partials, no need to create two. Our convention is to create a unique partial named `sqlite.md` which is going to hold the shared content between multiple OS.